Modix.Bot/Modules/IlModule.cs

```diff
@@ -115,7 +115,7 @@ private async Task<EmbedBuilder> BuildEmbedAsync(SocketGuildUser guildUser, stri
                .WithTitle("Decompile Result")
                .WithDescription(result.Contains("Emit Failed") ? "Failed" : "Successful")
                .WithColor(failed ? new Color(255, 0, 0) : new Color(0, 255, 0))
-               .WithAuthor(a => a.WithIconUrl(Context.User.GetDefiniteAvatarUrl()).WithName(guildUser?.Nickname ?? Context.User.Username));
```

Comment:
Why remove the icon here?

Reply:
If you just pass in an `IGuildUser`, it adds the icon for you.

BUT

1. The name is set to `$"{user.Nickname ?? user.Username}#{user.Discriminator}"`, which I thought was the opposite of what this PR aims to do.
2. You might want to test using someone that just has the default Discord logo avatar. I'm not sure it'll show up.

https://github.com/discord-net/Discord.Net/blob/dev/src/Discord.Net.Core/Extensions/EmbedBuilderExtensions.cs#L33-L34

Reply:
Ah, yeah, good point. This was actually one of the reasons I started investigating this, but ended up regressing it by accident when going through and refactoring. Will fix.